### ISSUE TITLE
Problem in parse when the string finalize with "\""

### ISSUE THREAD

**Issue by juniorgasparotto** [fclp/fluent-command-line-parser#65]
Hello,

I'm with a problem when my input is: "abc\"efd\"", I thing that output should be [abc"efd"](without []), but the output is [abc"efd] without the last char ' " '. It's very strange.

[]s

**Comment by kkm000** [comment]
This is how .NET own parser mangles args[], AFAIK. The only fix would be to parse off `Environment.CommandLine`.

**Comment by siywilliams** [comment]
Yes @kkm000 is correct in that .NET has already had a go at chewing that string value to pieces before it gets into the parser.

I just tried throwing the raw values directly into the parser `fclp.Parse(new []{ "--test", "abc\"efd\"" });` which worked as expected.
